.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -60,7 +121,26 @@ function commentPR(pr) {
         });
 }
 
-/**
- * Create comment on each pull request
- */
-prList.reduce((promise, pr) => promise.then(() => commentPR(pr)), Promise.resolve());
+const run = function () {
+    return Promise.all([
```

Comment:
Should this be `Promise.allSettled`?

Reply:
We need both promises to be successful otherwise it will not work. Thus `.all`

Reply:
Ah, gotcha. I got those two switched in my head 🙃